packages/components/src/notice/test/index.js

```diff
@@ -38,63 +34,47 @@ describe( 'Notice', () => {
 			</Notice>
 		);
 
-		expect( renderer.getRenderOutput() ).toMatchSnapshot();
+		expect( container ).toMatchSnapshot();
 	} );
 
 	it( 'should not have is-dismissible class when isDismissible prop is false', () => {
-		const renderer = new ShallowRenderer();
-
-		renderer.render( <Notice isDismissible={ false } /> );
+		const { container } = render( <Notice isDismissible={ false } /> );
 
-		const classes = new TokenList(
-			renderer.getRenderOutput().props.className
-		);
-		expect( classes.contains( 'components-notice' ) ).toBe( true );
-		expect( classes.contains( 'is-dismissible' ) ).toBe( false );
+		expect( container.firstChild ).toHaveClass( 'components-notice' );
```

Comment:
Ideally the notice should have some sort of a role (maybe `status` ) but `container.firstChild` is good enough for now if we don't refactor the component 👍

Reply:
That's probably too big of a change for test refactoring.